README.md

```diff
@@ -57,6 +57,8 @@ Web user interface for Batch Analysis
 
 ### Build and publish Docker image
 
+(You should not need to manually build or publish the image: it should happen automatically after your PR is merged to main).
+
 From the root of the repository run the below steps:
 1. To build the docker image run: `docker build . --no-cache -t us.gcr.io/broad-dsp-gcr-public/terra-batch-analysis-ui:<tag>`
```

Comment:
While you are changing this file, can you also change `terra-batch-analysis-ui` to `cbas-ui` here and below?